mods/website/web/style.css

```diff
@@ -271,7 +271,7 @@ section {
 
 
 .section1 .right{
-  flex:  1;
+  /*flex:  1;*/
```

Comment:
of this isn't needed anymore you could just delete it maybe

Reply:
I like to comment things out when I'm not 100% sure. Will delete on next edit of that file